docker-compose.yml

```diff
@@ -33,6 +33,10 @@ services:
       - POSTGRES_HOST=db
       - POSTGRES_PORT=5432
       - POSTGRES_DB=ft_transcendence
+      - FORTYTWO_APP_ID=${FORTYTWO_APP_ID:?required}
+      - FORTYTWO_APP_SECRET=${FORTYTWO_APP_SECRET:?required}
+      - SESSION_SECRET=${SESSION_SECRET:?required}
+      - MEMCACHED_SECRET=${MEMCACHED_SECRET:?required}
```

Comment:
I moved these environment variables here because usually, we run the project with Docker Compose, and we can define them with a `.env` file.

When we run the `transcendence-app` in standalone mode, for example, for tests or in Github CI, we use the `/transcendence-app/.env.{environment}` files

Reply:
I had an .env file from when we ran tests for the app in production mode, I didn't change it, and it worked. Should I erase all of the fields that you don't mention here?  Wouldn't that break the prod mode on my side?

Reply:
I think the `FORTYTWO_APP_CALLBACK_URL` for the development environment is `http://localhost:3000/api/v1/auth/login` and for production is `http://localhost/api/v1/auth/login` (different ports and possibly different domain/IP). The rest can be the same.